### DISCUSSION TITLE
Is it possible to set the timezone used on the dashboard?

### DISCUSSION THREAD

**Discussion by alexandergitter** [bensheldon/good_job#1028] [Q&A] [answered]
Timestamps in my database and in how the Rails app uses them are in UTC.
Both the chart on the GoodJob dashboard as well as the times shown in various places use UTC times. Is it possible to change the timezone that GoodJob uses to display times on its dashboard?

**Comment by bensheldon** [comment, marked as answer]
Interesting! There isn't currently a supported way to do that (maybe there should be), but here's a workaround:

```ruby
# config/initializers/good_job.rb
Rails.application.config.to_prepare do
  class GoodJob::ApplicationController < ActionController::Base
    CUSTOM_TIMEZONE = "America/Los_Angeles"

    around_action :set_custom_timezone

    def set_custom_timezone
      original_timezone = Time.zone
      Time.zone = CUSTOM_TIMEZONE
      yield
    ensure
      Time.zone = original_timezone
    end
  end
end
```

**Reply by alexandergitter** [reply]
Perfect, thank you :)

We have a way for users (including admins) to set their own time zone setting. Something like this should make it possible to integrate that with the GoodJob dashboard.
